extensions/Vendor/FRAUNHOFER_materials_pbr/README.md

```diff
@@ -0,0 +1,146 @@
+
+### Specular - Glossiness
+
+<img src="figures/specular_glossiness_2.png" align="middle" height="220" style="display: block; margin: 0 auto; padding: 20px 0 10px 0;">
+
+The specular-glossiness material model is defined by the following properties:
+* Diffuse:    Reflected diffuse color of the material
+* Specular:   Specular color of the material
+* Glossiness: Glossiniess of the material
+
+<p style="text-align:justify;">The diffuse value represents the reflected diffuse color of the material. Raw metals have a diffuse value of (0, 0, 0). The specular value defines specular reflectance at normal incidence (F0). The Glossiness value is a factor between 0.0 (rough surface) and 1.0 (perfectly smooth surface) and represents the surface irregularities that cause light diffusion. Figure 1 shows the three components of the specular-glossiness model and the rendered result.</p>
+
+The following table lists the allowed types and ranges for the specular-glossiness model:
+
+| Property | Type | Range | Default Value | Description |
+|:------------:|:----:|:-----:|:-:|:-----:|:-----------:|
+| `diffuseFactor`   | `FLOAT_VEC4` | [0, 1] for all components | [1,1,1,1] | The RGB components of the reflected diffuse color of the material. For raw metals the diffuse color is black (0.0). The fourth component (A) is the `opacity` of the material. |
+| `specularFactor`  | `FLOAT_VEC3` | [0, 1] for all components | [1,1,1]   | The specular RGB color of the material. |
+| `glossinessFactor`| `FLOAT`      | [0, 1]                    | 1         | The glossiness of the material surface (0 is glossiness, 1 is full glossiness). |
+| `diffuseTexture`  | string       | valid texture identifier  |           | Texture with RGB components of the reflected diffuse color of the material. For raw metals the diffuse color is black (0.0). If the fourth component (A) is present, it represents the `opacity` of the material. Otherwise, an `opacity` of 1 is assumed. |
+| `specularGlossinessTexture` | string       | valid texture identifier  | | RGBA texture, containing the specular color of the material (RGB components) and its glossiness (A component).|
+
+The factors (diffuseFactor, specularFactor, glossinessFactor) scale the components given in the respective textures (diffuseTexture, specularGlossinessTexture).
+If a texture is not given, all respective texture components are assumed to have a value of 1.
+
+For example, assume a value of `(1.0, 0.5, 0.5)` is read from an RGB `diffuseTexture`, and assume that `diffuseFactor` would be given as `(0.1, 1.0, 0.1, 1.0)`.
+Then, the result would be `(1.0 * 0.1, 0.5 * 1.0, 0.5 * 0.1, 1.0 * 1.0) = (0.1, 0.5, 0.05, 1.0)`;
+
+
+<strong>Usage Example:</strong>
+
+```javascript
+"materials": {    
+    "rough_gold2": {
+        "extensions": {
+            "KHR_materials_pbr" : {
+                "technique" : "PBR_specular_glossiness",
```

Comment:
Ah, I think I see now; this identifies the shading model.  The extension spec should explain this before showing the example.  It would also be useful to show texture references here so readers see a complete example.